ResearchKit/Common/ORKOrderedTask.m

```diff
@@ -283,7 +283,7 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
 static NSString * const ORKSpatialSpanMemoryStepIdentifier = @"cognitive.memory.spatialspan";
 static NSString * const ORKToneAudiometryPracticeStepIdentifier = @"tone.audiometry.practice";
 static NSString * const ORKToneAudiometryStepIdentifier = @"tone.audiometry";
-static NSString * const ORKDeviceMotionReactionTimeStepIdentifier = @"reactionTime.deviceMotion";
+static NSString * const ORKReactionTimeStepIdentifier = @"reactionTime.deviceMotion";
```

Comment:
Rename the identifier string to "reactionTime" to align with the const value name?